docs/external-content/btp-manager/docs/user/_sidebar.ts

```diff
@@ -0,0 +1,23 @@
+export default [
```

Comment:
Will this be hardcoded? If yes, then it would be good to configure it prettier because now it's really hard to read. If not, then it should be in `.gitignore`.